packages/fluentui/react-northstar/src/components/Carousel/CarouselNavigationItem.tsx

```diff
@@ -99,15 +104,17 @@ class CarouselNavigationItem extends UIComponent<WithAsProp<CarouselNavigationIt
         {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
         {...unhandledProps}
       >
-        {Box.create(indicator, {
-          defaultProps: () => ({
-            className: CarouselNavigationItem.slotClassNames.indicator,
-            styles: styles.indicator,
-          }),
-        })}
-        {Box.create(content, {
-          defaultProps: () => ({ as: 'span', styles: styles.content }),
-        })}
+        {!content &&
```

Comment:
why were these conditions added?

Reply:
Because otherwise would render the dots and the thumbnail side by side

Reply:
ok, then at least move this to a function and return either the content box or indicator box from there, conditionally based on content.

Reply:
Done